cmd/ci-operator-configresolver/main.go

```diff
@@ -106,6 +108,20 @@ func validateOptions(o options) error {
 	if o.cycle == 0 {
 		return fmt.Errorf("invalid cycle: duration cannot equal 0")
 	}
+	if o.configPath == "" {
+		return fmt.Errorf("--config is required")
+	} else {
+		if _, err := os.Stat(o.configPath); err != nil && os.IsNotExist(err) {
+			return fmt.Errorf("--config points to a nonexistent directory: %v", err)
+		}
+	}
+	if o.registryPath == "" {
+		return fmt.Errorf("--registry is required")
+	} else {
+		if _, err := os.Stat(o.registryPath); err != nil && os.IsNotExist(err) {
+			return fmt.Errorf("--registry points to a nonexistent directory: %v", err)
+		}
```

Comment:
Don't we already check this when creating the config/registry agents?

Reply:
Yeah but you get it as some semi-related error that's harder to interpret. This is useful for users since it tells them that the flag they passed was bad. Right now you see this:

```
FATA[0000] Failed to get registry agent: Failed to load registry: failed to load ci-operator registry (lstat : no such file or directory) 
```

Reply:
That's covered by the `if` part of the conditional, isn't it? That is fine.

If we get a bad error message when there's a problem reading the configuration/registry ("the file/directory doesn't exist" being one of the possibilities, which can also happen on reloads), we should fix that instead of adding this kind of code to every user.